tests/support_server/wait_children.pm

```diff
@@ -31,7 +31,7 @@ sub run {
     wait_for_children;
 
     unless (get_var('REMOTE_CONTROLLER')) {
-        send_key("ctrl-c");
+        type_string({text => '', terminate_with => 'ETX'});
```

Comment:
This will only work when `is_serial_terminal()` returns true. You still need to use `send_key` for VNC.

I'm not sure why it does not work when the virtio console is active. Perhaps you could try it without piping to `tee`. That is not really necessary when the serial terminal is active anyway. Also you could trying sending EOT instead or some other control character or try sending ETX twice.

Reply:
so I've tried with 
* sending ETX twice -> no success
* remove the `| tee` -> no success
* type_string '\cC' -> no success

the only thing which could work is to switch to another console which is not virtio, but this would invalidate the test I guess, as it states 
```
 # the supportserver image can be different version than the currently tested system
 34     # so try to login without use of needles
 35     select_virtio_console();

```

so I'm not sure how to proceed now...